runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -60,6 +61,7 @@
 )
 public class BarbarianAssaultPlugin extends Plugin
 {
+	static final String CONFIG_GROUP = "barbarianAssault";
```

Comment:
Reference this constant from the config class as well instead of having it hard-coded there also.